cypress/fixtures/selectors/index.js

```diff
@@ -57,4 +57,16 @@ export const bonuses = {
         getButton: '[aria-label="bonusButton"]', // кнопка "Получи бонус!"
         closeButton: '[class^="RegistrationBonus_root"] > header > button:nth-child(1)', // кнопка "Закрыть"
     },
-};
\ No newline at end of file
+};
+export const header = {
+    registrationButton: {
+        openButton: 'button[class="Button_root__4U57W HeaderClient_regBtn__q5UOv Button_root_primary___iDsV"]',
+        currencyList: '#downshift-1-toggle-button',
+        closeButton: '[class="HeaderModal_close___E9yg"]',
+    },
+    currency: {
+        USD: '#downshift-1-item-0',
+        RUB: '#downshift-1-item-1',
+        UZS: '#downshift-1-item-2',
+    },
```

Comment:
вяжемся к дата-элементу или классу:
```suggestion
currencyNameList: '[class^="CurrencySelect_item"]', // 16 валют из списка
```

Reply:
сделано